client/ayon_sitesync/plugins/publish/integrate_site_sync.py

```diff
@@ -35,21 +23,15 @@ def process(self, instance):
 
         context = instance.context
         project_name = context.data["projectEntity"]["name"]
-        modules_by_name = context.data["openPypeModules"]
-        sync_server_module = modules_by_name["sync_server"]
+        addons_by_name = context.data["ayonAddons"]
+        sitesync_addon = addons_by_name["sitesync"]
```

Comment:
```suggestion
        sitesync_addon = addons_by_name["sitesync"]
        if not sitesync_addon.enabled:
            return
```

Reply:
Why was marked as resolved?

Reply:
I think I updated it after first comment, updated it after later one.